docs/install.md

```diff
@@ -278,7 +278,7 @@ codacy-postgres      1/1    12d
     -   Enable persistence on `listener`
     -   Enable persistence on `minio`
     -   Setup resources and limits
-    -   Enable the Ingress on `codacy-api`
+    -   Configure the Ingress on `values-production.yaml`
```

Comment:
@rtfpessoa, could you please provide just a bit more detail?

Maybe something along the lines of "follow the instructions in the comments for the `codacy-ingress` section in the `values-production.yaml` file" would be enough?